jupyter_sphinx/ast.py

```diff
@@ -270,6 +303,9 @@ def cell_output_to_nodes(cell, data_priority, write_stderr, dir, thebe_config):
                 continue
             data = output["data"][mime_type]
             if mime_type.startswith("image"):
+                ####################################
+                # TODO: Figure out how to handle either inline or absolute image paths
```

Comment:
Which paths are you referring to over here? I thought that we're in full control of where we're writing the image here.

Reply:
I think this is a carry over from a previous bug? It works fine now in tests and such